kedro/extras/datasets/tracking/metrics_dataset.py

```diff
@@ -59,12 +59,14 @@ class MetricsDataSet(JSONDataSet):
 
     """
 
+    VERSIONED_FLAG_KEY = True
+
     # pylint: disable=too-many-arguments
     def __init__(
         self,
         filepath: str,
+        version: Version,
```

Comment:
Do we still need to overwrite `__init__` here? `JSONDataSet` has `version: Version = None` - not sure why this is now a compulsory argument here? 🤔

Reply:
Yea actually I forgot to put it back it. I was trying something else. This can be optional.